### ISSUE TITLE
how to modify hyperparameters since I need to augment data by rotation

### ISSUE THREAD

**Issue by frabob2017** [ultralytics/yolov5#9497]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I want to modify hyperparameters since I need to augment data by rotation, I found the default one has not set rotation. If I need to do it, I modify [hyp.scratch-low.yaml] directly or I need to create my own yaml file? 

Based on this instruction about hyperparameter optimization, I do not find specific instruction. there is an introduction of  --evolve there. I really do not understand it well. 
https://github.com/ultralytics/yolov5/issues/607

Hyperparameters. Default hyperparameters are in [hyp.scratch-low.yaml](https://github.com/ultralytics/yolov5/blob/master/data/hyps/hyp.scratch-low.yaml). We recommend you train with default hyperparameters first before thinking of modifying any. In general, increasing augmentation hyperparameters will reduce and delay overfitting, allowing for longer trainings and higher final mAP. Reduction in loss component gain hyperparameters like hyp['obj'] will help reduce overfitting in those specific loss components. For an automated method of optimizing these hyperparameters, see our [Hyperparameter Evolution Tutorial](https://github.com/ultralytics/yolov5/issues/607).

**Comment by glenn-jocher** [comment]
@frabob2017 not sure I understand your question. To update hyperparameters just update hyperparameters:
https://github.com/ultralytics/yolov5/blob/11640698977724daf7982c9da398c2ee2f2b6e91/data/hyps/hyp.scratch-low.yaml#L25

**Comment by frabob2017** [comment]
> @frabob2017 not sure I understand your question. To update hyperparameters just update hyperparameters:
> 
> https://github.com/ultralytics/yolov5/blob/11640698977724daf7982c9da398c2ee2f2b6e91/data/hyps/hyp.scratch-low.yaml#L25



> @frabob2017 not sure I understand your question. To update hyperparameters just update hyperparameters:
> 
> https://github.com/ultralytics/yolov5/blob/11640698977724daf7982c9da398c2ee2f2b6e91/data/hyps/hyp.scratch-low.yaml#L25

Thank you for your answer, I guess that I can modify hyp.scratch-low.yaml directly. Let me try and update it. It is a great supporting environment.
